app/code/Magento/Catalog/Model/Product/Attribute/Backend/TierPrice/SaveHandler.php

```diff
@@ -135,13 +135,11 @@ private function getAdditionalFields(array $objectArray): array
      * Check whether price has percentage value.
      *
      * @param array $priceRow
-     * @return int|null
+     * @return float
      */
-    private function getPercentage(array $priceRow): ?int
+    private function getPercentage(array $priceRow): float
```

Comment:
Will such type hinting work when there is `int`?

Reply:
Yes, of course.

Reply:
![image](https://user-images.githubusercontent.com/9420807/48942791-9c503580-ef29-11e8-80c2-06dbfa0f7104.png)

Reply:
Just checked :)

Reply:
@swnsma nice idea ;) such signature will convert to `float` though which is not desired: https://3v4l.org/eE28s

Please remove return type and make it possible to pass `int` unchanged.